API.md

```diff
@@ -1,41 +1,15 @@
 ## `fontnik`
 
-### `range(options: object, callback: function)`
+### `range(fontpath: string, start: number, end: number, callback: function)`
```

Comment:
I would kinda prefer leaving this as an options object, makes it a little easier to make additions to the API later without causing breaking changes.